RELEASE/scripts/autoscend/auto_familiar.ash

```diff
@@ -682,6 +682,16 @@ void preAdvUpdateFamiliar(location place)
 		}
 	}
 
+	if (my_familiar() == $familiar[Reagnimated Gnome])
+	{
+		if (!get_property("_auto_gnomeArenaVisited").to_boolean())
+		{
+			visit_url("arena.php");
+			set_property("_auto_gnomeArenaVisited", "true");
+		}
+		autoEquip($slot[familiar], $item[gnomish housemaid's kgnee]);
+	}
+
```

Comment:
I'd add a comment saying that this is turn free and can be done once a day so future generations of autoscend devs don't have to look it up on the wiki.

Reply:
> You need to run the assemble.sh file in the BUILD directory for that Shorter-Order Cook change to actually do something (and commit the updated data/autoscend_familiars.txt file that it will regenerate).

Thanks! I wasn't sure how that worked but now I know moving forward 👍